src/test/java/athleticli/commands/sleep/DeleteSleepCommandTest.java

```diff
@@ -0,0 +1,47 @@
+package athleticli.commands.sleep;
+
+import athleticli.data.Data;
+import athleticli.data.sleep.Sleep;
+import athleticli.data.sleep.SleepList;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeleteSleepCommandTest {
+
+    @Test
+    public void execute_validIndex_success() {
+        Data data = new Data();
+        SleepList sleepList = data.getSleeps();
+        sleepList.add(new Sleep("08:00", "10:00"));
```

Comment:
Consider using @BeforeEach to reduce code redundancy, otherwise nice testing of different cases

Reply:
Yes good suggestion!

Reply:
Will be implemented in the next iteration